verification/curator-service/ui/cypress/integration/components/LandingPage.spec.ts

```diff
@@ -218,7 +218,9 @@ describe('LandingPage', function () {
         cy.get('#password').type('tT$5aaaaak');
         cy.get('#passwordConfirmation').type('tT$5aaaaak');
         cy.get('#isAgreementChecked').check();
-        for (let i = 0; i < 5; i++) {
+        for (let i = 0; i < 7; i++) {
```

Comment:
why is the Sign-in button being clicked so many times?

Reply:
It is clicked so many times to trigger the login/register limiter that will be used after 4 unsuccessful attempts. I will change the number of clicks to the previous commit which is 5 times.